### PR TITLE
Update EIP-2537: Gas pricing pairing operations

### PR DESCRIPTION
**Pull request by MarekM25**
**Update:** Changed numbers to Paweł's proposition, details here: https://gist.github.com/MarekM25/79982c313c1179cbfd68c9414f8925fa

**ATTENTION: ERC-RELATED PULL REQUESTS NOW OCCUR IN [ETHEREUM/ERCS](https://github.com/ethereum/ercs)**

--

When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by garyschulte** [comment]
👍  from besu, implementing these gas costs and benchmarking on a nuc 13:
https://gist.github.com/garyschulte/c3e7a8015c8014707303e7f3a83dd117

**Comment by MarekM25** [comment]
> 👍 from besu, implementing these gas costs and benchmarking on a nuc 13: https://gist.github.com/garyschulte/c3e7a8015c8014707303e7f3a83dd117

@garyschulte I updated the numbers, please recheck and approve if Besu is fine with those numbers

File: EIPS/eip-2537.md [hunk 1]
```diff
@@ -287,7 +287,7 @@ Discounts table as a vector of pairs `[k, discount]`:
 
 #### Pairing check operation
 
-The cost of the pairing check operation is `43000*k + 65000` where `k` is a number of pairs.
+The cost of the pairing check operation is `32587*k + 37698` where `k` is a number of pairs.
```
Comment (chfast) [inline EIPS/eip-2537.md:290]: Maybe let's round these to 32600 and 37700.

Comment (MarekM25) [reply EIPS/eip-2537.md]: Done, thx @chfast :)

**Review by chfast — APPROVED** [review]

**Comment by mratsim** [comment]
As mentioned in https://github.com/ethereum/EIPs/pull/9145 2x pairing should have almost the same gas cost as the KZG_POINT_PRECOMPILE as it is doing the same thing, a pairing check with 2 pairs of (G1, G2) points.

EIP-4844 was priced with BLST and gnark.

I'm not sure what happened between the 1.8M gas (https://github.com/ethereum/EIPs/pull/4844/files#diff-c72ce77807acfea9e9693b7999f247841df938c9645be37013903805fa8b0c09R51) and the current 50k gas but I assume there are supporting benchmarks for both behind with hardware from 2022. cc @protolambda

**Review by ralexstokes — APPROVED** [review]
lgtm

**Pull request merged**